chapters/intro/main.tex

```diff
@@ -1,26 +1,165 @@
+Health Board (UHB), the hope was to apply some technique(s) from machine
+learning to reveal insights into their patient population within the hospital
+system. The journey to this point has not been without its deviations from this
+primary task, but that objective has been achieved.
+
+With an administrative dataset provided by \ctmuhb, exploratory analysis found
+that the population in question was deeply varied and heterogeneous --- as
+expected. However, the dataset was insufficiently detailed to construct
+meaningful models on the entire population or system, opening up an interest in
+population segmentation techniques. While data requests were made, the gathering
+of literature began. Even with the expansive nature of machine learning
+literature, two clear patterns emerged.
+
+First, the ethics of data and machine learning algorithms were being discussed,
+but the discourse appeared separately from the machine learning publications. In
+small amounts, the development of new, `fair' machine learning techniques
+appeared. These methods consider new formulations and objectives based on
+mathematical fairness, a concept either derived from or with common roots in
+game theory. Game theory is a branch of mathematics which applies rules and
+logic to resolve and analyse scenarios involving conflict, cooperation and
+competition. The reinventing of techniques and paradigms to be fair raised
+questions about how far an addition of some game theory could improve the
+performance of an existing algorithm, as opposed to creating an entirely new
+one.
+
+Second, the vast majority of publications that introduced a novel machine
+learning method would contain a boilerplate evaluation section. In each article,
+the proposed method would be pitted against a few of its contemporaries by
+comparing a handful of metrics on a handful of datasets. Typically, these
+metrics and datasets would be taken from a small pool which covered that
+technique; this makes sense. There are appropriate measures for various
+techniques, and it is important to make comparisons relative to a fixed point.
+However, this approach to algorithm evaluation exhibits two issues --- there is
+a lack of diversity in the resources used to assess algorithm performance, and
+there is some incongruence between the power of the evaluation process and the
+conclusions drawn from that process. Often, a method would be deemed
+`state-of-the-art' or `better' based on a process that offered little insight
+into the actual quality of that method. This reliance on a narrow assessment
+process prompted research into how else an algorithm could be evaluated
+objectively.
+
+Over the course of this project, it became clear that the requests for more
+detailed datasets would not pan out in time, and so the task became one of
+extracting as much actionable information from the data at hand. Simultaneously,
+\ctmuhb\ provided further administrative datasets which related to the members
```

Comment:
I think the style of these first 3 paragraphs is not right, things like "would not pan out in time" feel more reflective (and could perhaps be in your conclusion) but they should not be in your introduction. The introduction should be more upbeat and to the point:

- What's the problem?
- What's the solution approach?
- How are you going to write it out?